src/js/utils/hooks.js

```diff
@@ -42,3 +42,41 @@ export const useDidUpdateEffect = (onUpdate, deps) => {
         firstRef.current = true;
     }, deps);
 };
+
+export function getUrlSearchParams(key) {
+    const params = new URLSearchParams(window.location.search);
+    return params.get(key) || "";
+}
+
+export function updateUrlSearchParams(newTerm, key) {
+    const search = new URLSearchParams(window.location.search);
```

Comment:
You use `params` as the name for an object of the same type in the above function. Be consistent. I'd go with `params` as `search` implies action.